### PR TITLE
Dm plugin status

### PR DESCRIPTION
**Pull request by alexlarsson**
Add driver name and status details to docker info output

Here is some example output:

```
# docker info
Containers: 7
Images: 9
Driver: devicemapper
 Pool Name: docker-0:31-15505662-pool
 Data file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata file: /var/lib/docker/devicemapper/devicemapper/metadata
 Data Space Used: 2405.4 Mb
 Data Space Total: 102400.0 Mb
 Metadata Space Used: 2.3 Mb
 Metadata Space Total: 2048.0 Mb
Debug mode (server): true
Debug mode (client): false
Fds: 11
Goroutines: 11
LXC Version: 0.8.0
EventsListeners: 0
Kernel Version: 3.11.4-300.1.fc20.x86_64
```

### CODE REVIEW

**Comment by creack** [comment]
LGTM

File: devmapper/driver.go [hunk 1]
```diff
@@ -37,6 +37,21 @@ func (d *Driver) String() string {
 	return "devicemapper"
 }
 
+func (d *Driver) Status() [][2]string {
```
Comment (crosbymichael) [inline devmapper/driver.go:40]: @alexlarsson Is there a reason why you are using `[][2]stirng` instead of a map? `map[string]string` ?

**Comment by crosbymichael** [comment]
LGTM

File: devmapper/driver.go (hunk 1 above)
Comment (tianon) [inline devmapper/driver.go:40]: I wondered about that also, then realized it's probably for consistent ordering.

Comment (alexlarsson) [inline devmapper/driver.go:40]: Yes, its to get a static ordering in the CLI

**Pull request merged**